src/benchmarks/micro/corefx/System.Text.Json/Serializer/ReadJson.Dictionary.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public class ReadDictionary
+    {
+        private const string _jsonString = @"{""Hello"":""World"",""Hello2"":""World2""}";
+
+        [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+        [Benchmark]
+        public Dictionary<string, string> DeserializeDict()
+        {
+            return JsonSerializer.Parse<Dictionary<string, string>>(_jsonString);
+        }
+
+        [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+        [Benchmark]
+        public IDictionary<string, string> DeserializeIDict()
+        {
+            return JsonSerializer.Parse<IDictionary<string, string>>(_jsonString);
+        }
+
+        [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+        [Benchmark]
+        public IReadOnlyDictionary<string, string> DeserializeIReadOnlyDict()
```

Comment:
is `IReadOnlyDictionary` testing a different code path than `IDictionary` or `Dictionary`? If it's the same code path on the serializer side, I would stay only with `Dictionary`